include/definitions.h

```diff
@@ -23,9 +23,9 @@
 #define I2C_FREQ 100000 // 100kHz
 
 #define NUM_ADC_DEVICES 3
-#define I2C_ADDRESS_ADS1x15_0 0x4A
-#define I2C_ADDRESS_ADS1x15_1 0x48
-#define I2C_ADDRESS_ADS1x15_2 0x49
+#define I2C_ADDRESS_ADS1x15_0 0x48
+#define I2C_ADDRESS_ADS1x15_1 0// 0x49
+#define I2C_ADDRESS_ADS1x15_2 0x4B
 
```

Comment:
Does not look right to me, two devices with the same address, and the 0 is definitely wrong or will generate untested/unwanted side effects. We might want to temporarily reduce `NUM_ADC_DEVICES` to deal with a smaller number of the ADS installed

Reply:
resolve for this branch / PR